### PR TITLE
I3S-Debug - Rename ObbLayer -> BoundingVolumeLayer

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
1. Rename `ObbLayer` -> `BoundingVolume` layer
2. Naming refactoring

### CODE REVIEW

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 1]
```diff
@@ -342,6 +344,14 @@ export default class App extends PureComponent {
     }
   }
 
+  getBoundingVolumeColor(tile) {
+    const {boundingVolumeColorMode} = this.state.debugOptions;
+    const color =
+      this._colorsMap.getColor(tile, {coloredBy: boundingVolumeColorMode}) || DEFAULT_COLOR;
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:350]: Nit: `_colorMap`

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 2]
```diff
@@ -342,6 +344,14 @@ export default class App extends PureComponent {
...
+
+    return [...color, DEFAULT_BG_OPACITY];
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:352]: A bit risky if the color already has an opacity, you could get length 5?

Adding types and JSDoc could perhaps help a little.

File: examples/experimental/i3s-17-and-debug/color-map.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default class ColorMap {
     };
   }
 
-  getTileColor(tile, options) {
+  getColor(tile, options) {
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/color-map.js:36]: JSDoc headers on all methods please

File: examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js [hunk 4]
```diff
@@ -75,7 +75,7 @@ export default class TileLayer extends Tile3DLayer {
       });
 
     const color = colorsMap
-      ? colorsMap.getTileColor(tileHeader, {
+      ? colorsMap.getColor(tileHeader, {
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js:78]: Nit: colorMap

**Review by ibgreen — APPROVED** [review]

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js]: Renamed

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js:352]: Will Ad JSDoc in further PR. `getColor` method returns RGB format always.

File: examples/experimental/i3s-17-and-debug/color-map.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/color-map.js:36]: Will do in separate PR

File: examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js (hunk 4 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js]: Renamed

**Pull request merged**